packages/crawler/src/crawler-engine.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import Apify from 'apify';
+// @ts-ignore
+import * as cheerio from 'cheerio';
+import { Logger } from 'logger';
```

Comment:
I would delay adding loger; otherwise convert all `console.log()` to use logger.